### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devkiran**
## Description

Remove the instructions from README. Dub now use OAuth flow.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/dub-link-shortener/README.md [hunk 1]
```diff
@@ -3,7 +3,3 @@
 Dub is the open-source link management platform for modern marketing teams to create marketing campaigns, link sharing features, and referral programs.
 
 This extension allows you to shorten and retrieve your Dub links directly inside Raycast.
-
-# Instructions
-
-You will need the API key from Dub. To generate API key, follow [these instructions](https://dub.co/docs/api-reference/tokens).
```
Comment (jfkisafk) [inline extensions/dub-link-shortener/README.md:9]: Suggestion (non-blocking): Might want to add OAuth instructions (and removing earlier workspace key) + showcase end-2-end flow like this: https://www.raycast.com/koinzhang/github-gist

**Review by jfkisafk — APPROVED** [review]

File: extensions/dub-link-shortener/README.md (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/dub-link-shortener/README.md:9]: Sounds like a good idea, maybe it could be good to do that before merge?

Comment (jfkisafk) [reply extensions/dub-link-shortener/README.md:9]: @devkiran here is the gif you can save into `media/` dir for the e2e flow in readme:

Comment (devkiran) [reply extensions/dub-link-shortener/README.md:9]: @jfkisafk Sounds good. I've updated the README. Thank you

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**